app/code/Magento/Inventory/Model/Stock/Command/GetProductQuantityInStock.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\Stock\Command;
```

Comment:
I prefer to put this implementation upper level, 
as it's going to be one of the most-used services provided by Inventory 
(as all the front-end code needs Get Product Qty to render some data on front-end, make checkout etc.)

So, I propose to put it under this namespace:
`Magento\Inventory\Model`